docs/contribute/contribution-guidelines.md

```diff
@@ -4,113 +4,139 @@ title: Contribution guide
 
 ```text
-$ git remote add myprysmrepo https://github.com/<your_github_user_name>/prysm.git
+$ bazel test //beacon-chain/rpc/node:go_default_test --test_output=streamed --test_filter=TestNode_GetPeers
 ```
 
-Lastly, link your local clone to the Prysm repository to easily fetch future changes.
+For the list of all available flags to the `bazel test` command, you can see the reference documentation [here](https://docs.bazel.build/versions/master/command-line-reference.html#test).
+
+You can also run our full, end-to-end test suite with:
 
 ```text
-$ git remote add prysm https://github.com/prysmaticlabs/prysm.git
-$ git remote -v (you should see myrepo and prysm in the list of remotes)
+$ bazel test //endtoend:go_default_test --define=ssz=minimal
 ```
 
-Congratulations, you are now ready to begin contributing!
+### Running a Local Chain
 
-### **Making your first contribution**
+Although tests are the best and most simple way of ensuring your feature or bug fix contribution works, sometimes it can be helpful to run a real chain with your changes to ensure they are reflected at runtime. Here are some helpful commands to try running your own, local chain with 64 validators!
 
-Check out Prysm's [open issues on Github](https://github.com/prysmaticlabs/prysm/issues) and select one that you would like to work on, and then leave a comment to let the development team know. Alternatively, you can examine the code for areas that can be improved on and reach out to the development team to ask if they would like you to focus on it.
-
-Each time you begin a set of changes, ensure that you are working on a new branch that you have created as opposed to the `master` of your local repository. By keeping your changes segregated in this branch, merging your changes into the main repository later will be much simpler for the team.
+```
+$ bazel run //tools/genesis-state-gen --define=ssz=mainnet -- \
+  --num-validators=64 \
+  --output-ssz=/tmp/genesis.ssz \
+  --mainnet-config
+```
 
-To create a local branch for git to checkout, issue the command:
+Next up, run your beacon node with:
 
-```text
-$ git checkout -b feature-in-progress-branch
+```
+bazel run //beacon-chain --define ssz=mainnet -- \
+  --http-web3provider=http://localhost:7777 \
+  --web3provider=ws://localhost:7778 \
+  --bootstrap-node= \
+  --datadir /tmp/chaindata \
+  --force-clear-db \
+  --interop-genesis-state /tmp/genesis.ssz \
+  --interop-eth1data-votes \
+  --min-sync-peers=0 \
 ```
 
-To checkout a branch you have already created:
+In a separate window, run your validator with:
 
 ```text
-$ git checkout feature-in-progress-branch
+$ bazel run //validator --define=ssz=mainnet -- \
+  --keymanager=interop \
+  --keymanageropts='{"keys":64,"offset":0}' \
+  --disable-protect-attester \
+  --disable-protect-proposer \
+  --datadir /tmp/validator
 ```
 
-#### **Testing changes**
+### Making Your First Contribution
```

Comment:
Nitpick moment, headers below the high level `#` aren't capitalised on each letter :slightly_smiling_face: